tutorials/marble/jupyterlab-howto.md

```diff
@@ -0,0 +1,96 @@
+| ![Property Inspector Icon](images/jupyter/left-menu-property-inspector-icon.png) | Lets you look inside the tags of a Jupyter notebook                                                                                        |
+| ![Open Tabs Icon](images/jupyter/left-menu-open-tabs-icon.png)                   | Displays a list of tabs currently running.  Tabs can be shut down from here.                                                               |
+| ![Extension Manager Icon](images/jupyter/left-menu-extension-manager-icon.png)   | Displays any third-party extensions installed                                                                                              |
+
+
+## Creating Folders
+
+To create a folder click the `New Folder` button in the shortcut bar above the breadcrumb.
+
+This will create a new folder in the folder shown in the breadcrumb. 
+
+![Create Folder](images/jupyter/create-folder.png)
+
+## Upload Files
+
+To upload files click the `Upload Files` button in the shortcut bar above the breadcrumb.
+The file will be uploaded to the folder shown in the breadcrumb. 
+
+![Upload Files](images/jupyter/upload-files.png)
+
+## Refresh File List
+
+If a newly created folder or recently uploaded file is not seen in the list try refreshing it by click the `Refresh File List` button.
+
+![Refresh File List](images/jupyter/refresh-folder-list.png)
+
+
+## Git Clone
+
+![Git Clone](images/jupyter/git-clone.png)
+
+A shortcut for cloning a remote git repository.
+
+```{note}
+This button only becomes active when you are in a folder that does not contain a git repository.
+``` 
+
+
+## Folder Types
+
+When you click the `Folder` icon you will see there are some folders already created.  These are default folders in Marble that allow the user to store files and share files with other users.
+
+| Folder  Name                                                                     | Purpose                                                                                                                                               |
+|----------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| mypublic                                                                         | This is the user's public folder.  Files in this folder will be viewable by anyone in the node.                                                       |
+| public                                                                           | This shows the public folder of every user on the node.  <br> This folder is viewable by everyone and the files inside it can be accessed by everyone. |
```

Comment:
Let's say that these folders can be "viewed by anyone" instead of "accessed by anyone" just to be clear that they are read-only

Reply:
We should also talk about the relationship between "mypublic" and "public"... Anything that you put in "mypublic" will show up in a folder with your username in the "public" folder.

Reply:
- Changed the wording to 'viewable by everyone'
- Changed the wording to specifically mention that files put in the 'mypublic' folder will be vieable by anyone through the 'public' folder